### DISCUSSION TITLE
Do Tasmota Energy Plugs sum up total consumed/produced energy

### DISCUSSION THREAD

**Discussion by ilu2k21** [arendst/Tasmota#19491] [Q&A]
HI all,

i got some NousA1T in combination with homeassistant.
Every day the Energy total is resetted and starts from zero. Is this behavior normal?
Some other plugs i have sum up all the energy.

**Comment by sfromis** [comment]
When things work normally, the total is summed up, but only saved when settings are saved for other reasons, and at midnight. If you cut input power, consumption data since last save is lost.

**Reply by ilu2k21** [reply]
Ok, but normally the total which has been saved from days before  should persist?
Because after the last time i unplugged it it lost all data.

**Reply by sfromis** [reply]
Well, then things are not "working normally", and some debugging of what happens is relevant.
